CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+# Contributing
+We are seeking contributors to expand the feature set of cardanosharp-wallet to broaden the Cardano developer ecosystem. Please get in touch with the current list of maintainers if you want to help.
+
+## Maintainers
+ - Keith (keith@lovelace.academy)
+ - Kyle (kjohns1337@gmail.com)
```

Comment:
@Daniel-SchaeferJ do you want to add your email here?
@safestak-keith do you want your email here?